src/spork.h

```diff
@@ -120,23 +159,84 @@ class CSporkManager
         // we don't serialize private key to prevent its leakage
     }
 
+    /**
+     * Clear is used to clear all in-memory active spork messages. Since spork
+     * public and private keys are set in init.cpp, we do not clear them here.
+     *
+     * This method was introduced along with the spork cache.
+     */
     void Clear();
     void CheckAndRemove();
 
+    /**
+     * ProcessSpork is used to handle the 'getsporks' and 'spork' p2p messages.
+     *
+     * For 'getsporks', it sends active sporks to the requesting peer. For 'spork',
+     * it validates the spork and adds it to the internal spork storage and
+     * performs any necessary processing.
+     */
     void ProcessSpork(CNode* pfrom, const std::string& strCommand, CDataStream& vRecv, CConnman& connman);
+
+    /**
+     * ExecuteSpork is used to perform certain actions based on the spork value.
+     * Currently only used with Spork 12.
+     */
     void ExecuteSpork(int nSporkID, int nValue);
+
+    /**
+     * UpdateSpork is used by the spork RPC command to set a new spork value, sign
+     * and broadcast the spork message.
+     */
     bool UpdateSpork(int nSporkID, int64_t nValue, CConnman& connman);
 
+    /**
+     * IsSporkActive returns a bool for "on/off" sporks, which should be used to
+     * determine whether the spork can be considered active or not.
```

Comment:
Should probably clarify that this is only true for time-based sporks (which are majority), for pure value-based sporks like SPORK_5_INSTANTSEND_MAX_VALUE this function makes no sense and should not be used.